slices/logrotate.yaml

```diff
@@ -0,0 +1,29 @@
+package: logrotate
+
+essential:
+  - logrotate_copyright
+
+slices:
+  bins:
+    essential:
+      - cron_bins
```

Comment:
Hey, is it removed though?

Reply:
the logrotate package dependency itself was removed from erlang and/or rabbitmq-server slice. Same for cron. But I left both logrotate and cron slice definitions in case they are needed in the future as dependencies for other slice definitions.